README.md

```diff
@@ -1,1218 +1,533 @@
+
+### Prerequisites
+
+- C compiler (C99 compatible)
+- Make
 - libtool
-- make
-- mingw64-x86_64-dlfcn
-- mingw64-x86_64-gcc-core
-- mingw64-x86_64-zlib
 - pkg-config
-
-To build, just run the `build.sh` script in your Cygwin terminal. By default, it installs
-to "C:/Program Files/collectd". You can change the location by setting the INSTALL_DIR
-variable:
-
-$ export INSTALL_DIR="C:/some/other/install/directory"
-$ ./build.sh
-
-or:
-
-$ INSTALL_DIR="C:/some/other/install/directory" ./build.sh
-
-
-Crosscompiling
---------------
-
-  To compile correctly collectd needs to be able to initialize static
-  variables to NAN (Not A Number). Some C libraries, especially the GNU
-  libc, have a problem with that.
-
-  Luckily, with GCC it's possible to work around that problem: One can define
-  NAN as being (0.0 / 0.0) and `isnan` as `f != f`. However, to test this
-  "implementation" the configure script needs to compile and run a short
-  test program. Obviously running a test program when doing a cross-
-  compilation is, well, challenging.
-
-  If you run into this problem, you can use the `--with-nan-emulation`
-  configure option to force the use of this implementation. We can't promise
-  that the compiled binary actually behaves as it should, but since NANs
-  are likely never passed to the libm you have a good chance to be lucky.
-
-  Likewise, collectd needs to know the layout of doubles in memory, in order
-  to craft uniform network packets over different architectures. For this, it
-  needs to know how to convert doubles into the memory layout used by x86. The
-  configure script tries to figure this out by compiling and running a few
-  small test programs. This is of course not possible when cross-compiling.
-  You can use the `--with-fp-layout` option to tell the configure script which
-  conversion method to assume. Valid arguments are:
-
-    * `nothing`    (12345678 -> 12345678)
-    * `endianflip` (12345678 -> 87654321)
-    * `intswap`    (12345678 -> 56781234)
-
-
-Contact
--------
-
-  Please use GitHub to report bugs and submit pull requests:
-  <https://github.com/collectd/collectd/>.
-  See CONTRIBUTING.md for details.
-
-  For questions, development information and basically all other concerns please
-  send an email to collectd's mailing list at
-  (list at collectd.org).
-
-  For live discussion and more personal contact visit us in IRC, we're in
-  channel #collectd on freenode.
-
-
-Author
-------
-
-  Florian octo Forster (octo at collectd.org),
-  Sebastian tokkee Harl (sh at tokkee.org),
-  and many other [authors](AUTHORS) and [contributors](https://github.com/collectd/collectd/graphs/contributors).
+- POSIX threads (pthread) library
+
+### Building from Source
+
+```bash
+git clone https://github.com/collectd/collectd.git
+cd collectd
+./build.sh
+./configure
+make
+sudo make install
+```
+
+### Package Managers
+
+```bash
+# Debian/Ubuntu
+apt-get install collectd
+
+# RHEL/CentOS
+dnf install collectd
```

Comment:
`apt-get` is (nowadays) intended for things like scripts etc as it offers a bit more control over installation, but end-users are supposed to be using `apt` (it covers also functionality provided by other apt-* tools). And installs need root:
```suggestion
sudo apt install collectd

# RHEL/CentOS
sudo dnf install collectd
```

No idea what brew needs.